src/test/java/com/adyen/ApplicationInfoTest.java

```diff
@@ -28,4 +28,20 @@ public void TestApplicationInfoPrefilledLibraryFields() {
         assertNotNull(applicationInfo.getAdyenLibrary().getVersion());
         assertEquals(LIB_VERSION, applicationInfo.getAdyenLibrary().getVersion());
     }
+
+    @Test
+    public void TestApplicationInfoPrefilledSaleToAcquirerDataModel() {
+
+        SaleToAcquirerDataModel saleToAcquirerDataModel = new SaleToAcquirerDataModel();
+        String applicationInfoExpected = "{\n"
+                + "  \"applicationInfo\": {\n"
+                + "    \"adyenLibrary\": {\n"
+                + "      \"name\": \""+LIB_NAME+"\",\n"
+                + "      \"version\": \""+LIB_VERSION+"\"\n"
+                + "    }\n"
+                + "  }\n"
+                + "}";
```

Comment:
I would prefer something that's clearer than a concatenation. Could be either an external JSON file or  converting a plain dictionary into JSON?

Reply:
this is to keep it readable. We could do map to json or something.